melodic/distribution.yaml

```diff
@@ -6184,6 +6184,22 @@ repositories:
       url: https://github.com/milvusrobotics/mrp2_common.git
       version: melodic-devel
     status: maintained
+  mrp2_robot:
+    doc:
+      type: git
+      url: https://github.com/milvusrobotics/mrp2_robot.git
+      version: melodic-devel
+    release:
+      packages:
+      - mrp2_bringup
+      - mrp2_display
+      - mrp2_hardware
+      - mrp2_robot
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/milvusrobotics/mrp2_robot-release.git
+      version: 0.2.5-1
+    status: maintained
```

Comment:
In Indigo, this also had a `source` block.  It's not required for merging, but would you like to add it here?

Reply:
Sorry for late response. Closing the request for a new one.